tests/specs/bench/filter_group_header/main.out

```diff
@@ -7,7 +7,7 @@ benchmark      time (avg)        iter/s             (min … max)       p75
 --------------------------------------------------------------- -----------------------------
 
 group G1
-G1-B          [WILDCARD]
+G1-B         [WILDCARD]
 
 group G2
-G2-B          [WILDCARD]
\ No newline at end of file
+G2-B         [WILDCARD]
```

Comment:
If the number was big enough or enough decimal places were shown then this would fail.